ssh-pairing-server.c

```diff
@@ -13,16 +13,45 @@
 
 #define MAX_KEY_COUNT 10
 
+/*
+ * processing argv and providing a minimal usage / --help message describing
+ * what this program does would be helpful for any production level tool.
+ */
 int main(int argc, char *argv[])
 {
 	ssh_bind bind = ssh_bind_new();
 
 	// If a file doesn't exist it's ignored. If no host keys are available,
 	// ssh_bind_listen will fail immediately with a helpful error.
+	/*
+	 * this mimics the current default host key settings for sshd.
+	 *
+	 * if these should ever change then we get an inconsistency. would it
+	 * be possible to query these paths from libssh?
+	 *
+	 * furthermore, what if in sshd_config different settings are found? I
```

Comment:
That's a pitty. So the pairing-server is not a full drop-in replacement for sshd. And this can have undesirable side effects for non-default configurations.

Reply:
Yeah, but those are rare, especially when the system is still being set up and there was no opportunity for the user to change anything yet.

I've got to file the libssh issue upstream still, once it's fixed it'll "just work" (tm)

Reply:
> I've got to file the libssh issue upstream still, once it's fixed it'll "just work" (tm)

https://gitlab.com/libssh/libssh-mirror/-/issues/234